test/drag_test_basic.js

```diff
@@ -29,8 +29,34 @@ QUnit.test("Drag Item Upward", 2, function () {
 	});
 
 	testFrame.height = 350;
-	testFrame.src = 'testpages/dragdrop.html';
+	testFrame.src = 'testpages/html5_dragdrop.html';
 
 });
 
+QUnit.test("Drag Item Downward jQuery", 1, function () {
+	stop();
+
+	var testFrame = document.getElementById('pageUnderTest');
+	
+	testFrame.addEventListener('load',  function loadListener(){
+		testFrame.removeEventListener('load', loadListener);	
+		var pageUnderTest = document.getElementById('pageUnderTest').contentDocument.querySelector('body');
+		var draggable = pageUnderTest.querySelector('#draggable');
+		var target = pageUnderTest.querySelector('#dropTarget');
+		
+		// Timeout is required because jQuery isn't ready yet on the target page
+		setTimeout(function () {
+			syn.drag(draggable, {to: target}, function () {
+				var check = pageUnderTest.querySelector('#draggable')
+				ok(check.classList.contains('leftOfMark'), "jQuery drag did not register.");
+				
+				start();
+			});
+		}, 500);			
```

Comment:
Hm, I thought the `load` event only fired when _everything_ had been loaded…

Reply:
I agree that its probably fired when all of the resources had loaded, but once it loads, the javascript must execute and alter the element in some way.

I don't want to give the wrong impression that I know how this works, js being single-threaded and all, but cloud the load event execution halt the jquery-ui wiring-up execution?

All I know is: with the pause, this works for me 10/10, without the pause it fails 10/10

Reply:
Yeah, I don’t think the load event should fire until after the jQuery script has executed in the frame, so maybe there’s something else that has to be waited on?